### PR TITLE
Event sources detail panel

### PR DESCRIPTION
**Pull request by sturmer-eha**

### CODE REVIEW

File: client/controllers/articles.coffee [hunk 1]
```diff
@@ -71,22 +86,44 @@ Template.articles.events
...
+    event.preventDefault()
+    templateInstance.selectedSourceId.set @_id
+  "click .open-source-form": (event, template) ->
+    Modal.show("sourceModal", {userEventId: template.data.userEvent._id})
 
 Template.articleSelect2.onDestroyed ->
   templateInstance = Template.instance()
   templateInstance.$("#" + templateInstance.data.selectId).select2("destroy")
+
+Template.articleSelect2.helpers
```
Comment (nathanathan) [inline client/controllers/articles.coffee]: I think this was accidentally reintroduced. The onRendered function is now used to initialize the select2.

File: imports/ui/helpers.coffee [hunk 2]
```diff
@@ -37,4 +42,10 @@ UI.registerHelper 'incidentToText', (incident)->
       @locations.map(formatLocation).slice(0, -1).join(", ") +
       ", and " + formatLocation(@locations.slice(-1)[0])
     )
-  return "#{incidentDescription} in #{formattedLocations} #{formatTimeRange(@timeRange)}"
+  result = "#{incidentDescription} in #{formattedLocations}"
+  if @timeRange
```
Comment (nathanathan) [inline imports/ui/helpers.coffee]: With the addition of date ranges in a recent PR, this variable should be "dateRange" and the function should now be "formatDateRange"

File: client/controllers/articles.coffee (hunk 1 above)
Comment (sturmer-eha) [inline client/controllers/articles.coffee]: Good catch, thanks.

**Comment by sturmer-eha** [comment]
Should be good to merge now, I believe.

**Comment by nathanathan** [comment]
👍

**Pull request merged**